pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -726,30 +759,30 @@ func TestGetVerifiedPipelineFunc_GetFuncError(t *testing.T) {
 	}
 
 	testcases := []struct {
-		name                     string
-		requester                *test.Requester
-		resourceVerificationMode string
-		pipelinerun              v1beta1.PipelineRun
-		expectedErr              error
+		name                      string
+		requester                 *test.Requester
+		verificationNoMatchPolicy string
+		pipelinerun               v1beta1.PipelineRun
+		expectedErr               error
 	}{
 		{
-			name:                     "get error when oci bundle return error",
-			requester:                requesterUnsigned,
-			resourceVerificationMode: config.EnforceResourceVerificationMode,
-			pipelinerun:              *prBundleError,
-			expectedErr:              fmt.Errorf(`failed to get pipeline: failed to get keychain: serviceaccounts "default" not found`),
+			name:                      "get error when oci bundle return error",
+			requester:                 requesterUnsigned,
+			verificationNoMatchPolicy: config.FailNoMatchPolicy,
```

Comment:
Is the verificationnomatchpolicy relevant for these test cases, since they have matching policies?